src/api.jl

```diff
@@ -2787,7 +2787,9 @@ Returns the error message corresponding to the last error encountered by a libra
 
 """
 function getlasterror(prob::XpressProblem)
-    @invoke Lib.XPRSgetlasterror(prob, _)::String
+    out = Cstring(pointer(Array{Cchar}(undef, 512)))
+    s = Lib.XPRSgetlasterror(prob, out)
+    s == 0 ? unsafe_string(out) : "Unable to get last error"
 end
```

Comment:
This should probably be 
```Julia
function getlasterror(prob::XpressProblem)
    @invoke Lib.XPRSgetlasterror(prob, _)::String
    buffer = Array{Cchar}(undef, 512)
    buffer_p = pointer(buffer)
    GC.@preserve buffer begin
        s = Lib.XPRSgetlasterror(prob, buffer_p)
        return s == 0 ? unsafe_string(buffer_p) : "Unable to get last error"
    end
 end
```

Reply:
I think you are right! I've made this change along with another potentially similar issue in https://github.com/jump-dev/Xpress.jl/pull/126/commits/8833da4ddfbb87ebade64a71fcf1939ba9a04a97.

Reply:
Added more changes in here: https://github.com/jump-dev/Xpress.jl/pull/126/commits/d3fe3421dc420bfea015e389dbd03cb34cf05de9